.github/workflows/release.yaml

```diff
@@ -4,7 +4,12 @@ on:
   push:
     branches:
       - master
-  workflow_dispatch: ~
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'Hva skal vi kalle denne releasen?'
+        required: true
+        default: "SNAPSHOT"
```

Comment:
Er vel litt rart å ha SNAPSHOT som default på en release, selv om det er snakk om manuell trigging?

Reply:
Akkurat dette er litt work in progress. Fordi den eneste måten (som jeg vet om) å teste en workflow er å kjøre den i GA, så går det litt tregt å få dette riktig. I utgangspunktet ville jeg ha branch-name her men det er visst ikke trivielt.. 
Jeg tenker det viktigste er at det kommer klart frem at det ikke er en offisiell release, og om du skal bruke workflow_dispatch til å lage en offisiell release er det bare å endre.